index.d.ts

```diff
@@ -0,0 +1,29 @@
+interface UrlBuilderOptions {
+  allowUpscale: boolean,
+  backgroundColor: string,
+  borderRadius: string,
+  borderWidth: string,
+  circularMask: string,
+  color: string,
+  crop: string,
+  cropWithDifferentFocus: boolean,
+  disableWebP: boolean,
+  height: string,
+  flipHorizontally: boolean,
+  flipVertically: boolean,
+  forceDownload: boolean,
+  forceLossyCompression: boolean,
+  forceScaling: boolean,
+  imageFormat: string,
+  losslessCompression: string,
+  lossyCompressionLevel: string,
```

Comment:
```suggestion
  lossyCompressionLevel: number,
```
A percentage between 0 and 100

Reply:
Not sure how to represent this.